### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Note: I expect to take feedback and will fix the commit(s) to conform to the requested style.

Fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/firecamp/actions/runs/6344055793/attempts/1#summary-17233312792

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/firecamp/actions/runs/6344056064/attempts/1#summary-17233313532

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 <p align="center">
    <a href="https://discord.gg/8hRaqhK"><img alt="Discord online members" src="https://badgen.net/discord/members/8hRaqhK?color=5865F2&label=Discord&style=for-the-badge" /></a>
-   <a href="https://github.com/firecamp-dev/firecamp/stargazers"><img src="https://img.shields.io/github/stars/firecamp-dev/firecamp" alt="Github Stars"></a>
+   <a href="https://github.com/firecamp-dev/firecamp/stargazers"><img src="https://img.shields.io/github/stars/firecamp-dev/firecamp" alt="GitHub Stars"></a>
```
Comment (jsoref) [inline README.md:14]: brand

File: platform/firecamp-platform/src/components/modals/auth/Auth.sass [hunk 2]
```diff
@@ -19,7 +19,7 @@
     font-size: 16px
     text-transform: uppercase
 
-.authentication-modal-devider:before
+.authentication-modal-divider:before
```
Comment (jsoref) [inline platform/firecamp-platform/src/components/modals/auth/Auth.sass:22]: hopefully harmless?

File: platform/firecamp-ui/src/components/auth-panel/auths/Aws.tsx [hunk 3]
```diff
@@ -39,7 +39,7 @@ const Aws: FC<IAws> = ({ auth, onChange = () => {} }) => {
           dirtyInputs[input.id] &&
           !auth?.[input.id as keyof IAuthAwsV4]?.length
         ) {
-          errorMsg = `${input.name} can not be empty`;
+          errorMsg = `${input.name} cannot be empty`;
```
Comment (jsoref) [inline platform/firecamp-ui/src/components/auth-panel/auths/Aws.tsx:42]: https://www.grammarly.com/blog/open-and-closed-compound-words/

File: platform/firecamp-ui/src/components/dropdown/Dropdown.stories.tsx [hunk 4]
```diff
@@ -86,7 +86,7 @@ const Template = (args) => {
           }]
         }
         onSelect={item => {
-          setSelected(item.name || 'oopss...')
+          setSelected(item.name || 'oops...')
```
Comment (jsoref) [inline platform/firecamp-ui/src/components/dropdown/Dropdown.stories.tsx:89]: some projects are intentionally cute. this one doesn't really appear to be.

File: platform/firecamp-ui/src/components/editors/cm-gql-editor/CMGQueryEditor.interfaces.ts [hunk 5]
```diff
@@ -1,5 +1,5 @@
 //@ts-ignore  // TODO: install graphql as devDep type package
-import { GraphQLSchema } from 'graghql';
+import { GraphQLSchema } from 'graphql';
```
Comment (jsoref) [inline platform/firecamp-ui/src/components/editors/cm-gql-editor/CMGQueryEditor.interfaces.ts:2]: ?!

File: platform/firecamp-ui/src/components/editors/monaco-v2/SingleLineEditor.tsx [hunk 6]
```diff
@@ -354,7 +354,7 @@ const SingleLineEditor: FC<IEditor & TSLEditor> = ({
   value = type === 'number' ? '' + value : value;
   /**
    * 1. Check if number or not, if number then convert to string and show
-   * 2. Convert multiline string in to single line
+   * 2. Convert multiline string into single line
```
Comment (jsoref) [inline platform/firecamp-ui/src/components/editors/monaco-v2/SingleLineEditor.tsx:357]: changes for `into` have been quite controversial.

File: platform/firecamp-ui/src/components/grid/Column.tsx [hunk 7]
```diff
@@ -1,4 +1,4 @@
-//@ts-nocheckh
+//@ts-nocheck
```
Comment (jsoref) [inline platform/firecamp-ui/src/components/grid/Column.tsx:1]: ??

File: platform/firecamp-ui/src/components/response/tabs/Tabs.tsx [hunk 8]
```diff
@@ -138,7 +138,7 @@ const Tabs: FC<IResTabs> = ({
 export default Tabs;
 
 const ScriptErrors = ({ errors = [] }) => {
-  console.log(errors, 'erros..... errors');
+  console.log(errors, 'errors..... errors');
```
Comment (jsoref) [inline platform/firecamp-ui/src/components/response/tabs/Tabs.tsx:141]: ?

File: platform/firecamp-ui/src/components/table/primitive/table.interfaces.ts [hunk 9]
```diff
@@ -3,7 +3,7 @@ import { FocusEventHandler, MouseEventHandler, ReactNode } from 'react';
 
 /** Table options */
 export interface ITableOptions {
-  /** disabled column's keys. One can not edit within the column included in to disabledColumns */
+  /** disabled column's keys. One cannot edit within the column included in disabledColumns */
```
Comment (jsoref) [inline platform/firecamp-ui/src/components/table/primitive/table.interfaces.ts:6]: atypical change from `in to`

File: playgrounds/firecamp-graphql/src/store/slices/change.slice.ts [hunk 10]
```diff
@@ -29,7 +29,7 @@ interface IRequestChangeStateSlice {
    */
   disposeRCS: () => void;
 }
-//@note; always use _cloneDeep at its usage otherrwise it's value will be manipulate at global scope
+//@note; always use _cloneDeep at its usage otherwise its value will be manipulate at global scope
```
Comment (jsoref) [inline playgrounds/firecamp-graphql/src/store/slices/change.slice.ts:32]: this is a pair of unrelated corrections

`it's` = `it is`
`its` = possessive 
https://www.dictionary.com/e/its-vs-its/

File: playgrounds/firecamp-socket-io/src/components/connection-panel/connection/playground/BodyControls.tsx [hunk 11]
```diff
@@ -37,7 +37,7 @@ const BodyControls = ({ tabId = '', path = '', addNewEmitter = () => {} }) => {
           </div>
         </StatusBar.PrimaryRegion>
         <StatusBar.SecondaryRegion>
-          {/* <ShortcutsInfo tidabId={tabId} /> */}
+          {/* <ShortcutsInfo tabId={tabId} /> */}
```
Comment (jsoref) [inline playgrounds/firecamp-socket-io/src/components/connection-panel/connection/playground/BodyControls.tsx:40]: ?

File: playgrounds/firecamp-socket-io/src/store/slices/handleConnectionExecutor.slice.ts [hunk 12]
```diff
@@ -89,7 +89,7 @@ const createHandleConnectionExecutor: TStoreSlice<
       // connect
       executor.connect();
 
-      // set executor in to playground
+      // set executor in playground
```
Comment (jsoref) [inline playgrounds/firecamp-socket-io/src/store/slices/handleConnectionExecutor.slice.ts:92]: atypical `in to` change

File: platform/firecamp-ui/src/components/dropdown/Dropdown.stories.tsx (hunk 4 above)
Comment (Nishchit14) [reply platform/firecamp-ui/src/components/dropdown/Dropdown.stories.tsx:89]: Ha ha... I am trying to make it cute. :D

File: playgrounds/firecamp-socket-io/src/components/connection-panel/connection/playground/BodyControls.tsx (hunk 11 above)
Comment (Nishchit14) [reply playgrounds/firecamp-socket-io/src/components/connection-panel/connection/playground/BodyControls.tsx:40]: nice catch, thank you 🥇

**Comment by Nishchit14** [comment]
thank you @jsoref for taking the time to review the repo and fixing misspells and chores. i appreciate it a lot. i will be traveling this week frequently so it might take time to review and merge. but i am eager to merge it asap. thank you again!!

File: platform/firecamp-ui/src/components/dropdown/Dropdown.stories.tsx (hunk 4 above)
Comment (jsoref) [reply platform/firecamp-ui/src/components/dropdown/Dropdown.stories.tsx:89]: Happy to drop if you like...

**Review by Nishchit14 — APPROVED** [review]
LGTM :)

thanks for all these corrections.

**Comment by Nishchit14** [comment]
@jsoref how about to set up this [check-spelling](https://github.com/marketplace/actions/check-spelling) action for the firecamp repo? i think you just need to open a PR from your fork, right? also is it worth to run this action on all commits/PRs?

**Comment by jsoref** [comment]
Yes. I generally recommend that.

**Pull request merged**